### PR TITLE
xmpp-bridge: init at 0.6.0

### PR DESCRIPTION
**Pull request by Gigahawk**
## Description of changes

A tool that forwards the stdout of a command to an XMPP account.
https://github.com/majewsky/xmpp-bridge

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -6497,6 +6497,12 @@
     githubId = 127353;
     name = "Geoffrey Huntley";
   };
+  gigahawk = {
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:6500]: SPLIT

File: pkgs/by-name/xm/xmpp-bridge/package.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    libstrophe
+  ];
+
+  # Makefile is hardcoded to install to /usr
```
Comment (AndersonTorres) [inline pkgs/by-name/xm/xmpp-bridge/package.nix:27]: It should be easier to "manually" copy the files, then.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (Gigahawk) [reply maintainers/maintainer-list.nix:6500]: As in split the maintainers edit into a new commit?

Comment (AndersonTorres) [reply maintainers/maintainer-list.nix:6500]: yep

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Comment by Gigahawk** [comment]
@AndersonTorres thanks for the cleanup, build still works fine.

File: pkgs/by-name/xm/xmpp-bridge/package.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  nativeBuildInputs = [
+    installShellFiles
+    pkg-config
+  ];
+
+  buildInputs = [
+    libstrophe
+  ];
+
+strictDeps  = true;
```
Comment (AndersonTorres) [inline pkgs/by-name/xm/xmpp-bridge/package.nix:29]:
```suggestion
  strictDeps  = true;
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by AndersonTorres — APPROVED** [review]

File: pkgs/by-name/xm/xmpp-bridge/package.nix [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    runHook  preInstall
+
+    install -D -m 0755 build/xmpp-bridge "$out/bin/xmpp-bridge"
+    installManPage xmpp-bridge.1
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Connect command-line programs to XMPP ";
```
Comment (h7x4) [inline pkgs/by-name/xm/xmpp-bridge/package.nix:42]: nit:

```suggestion
    description = "Connect command-line programs to XMPP";
```

Comment (AndersonTorres) [reply pkgs/by-name/xm/xmpp-bridge/package.nix:42]: Not nit, but OK.

**Comment by h7x4** [comment]
@Gigahawk I think the only thing left is to amend the last commit into the previous one

**Pull request merged**